packages/accordion/package.json

```diff
@@ -30,10 +31,10 @@
         "url": "https://github.com/bring/hedwig-react-components/issues"
     },
     "dependencies": {
-        "@fortawesome/pro-solid-svg-icons": "^6.2.0",
+        "@fortawesome/fontawesome-svg-core": "^6.4.0",
```

Comment:
🗒️ The package `@fortawesome/fontawesome-svg-core` has to be explicitly added to the dependencies
https://fontawesome.com/docs/web/use-with/react/#_1-add-svg-core